### PR TITLE
Readme shorted. Docs referenced

### PR DESCRIPTION
**Pull request by jakethekoenig**
## Pull Request Checklist
- [x] Documentation has been updated, or this change doesn't require that

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -173,4 +89,4 @@ The goal for Mentat is to become the best way to write code. Some big improvemen
 
 If this is something you'd like to work on, jump right in! And if you want to join the team to work on this full time, message [@bio_bootloader](https://twitter.com/bio_bootloader) on twitter.
 
-To find something specific to work on, take a look at [open issues](https://github.com/AbanteAI/mentat/issues) and/or check this Github Project: [Mentat Development](https://github.com/users/biobootloader/projects/2)
+To find something specific to work on, take a look at [open issues](https://github.com/AbanteAI/mentat/issues).
```
Comment (jakethekoenig) [inline README.md:92]: It doesn't seem like we really use the project so I thought it was better to remove the link.

File: README.md [hunk 2]
```diff
@@ -58,10 +59,6 @@ cd mentat
 pip install -e .
 ```
 
-## Selecting which LLM Model to use
-
-We highly recommend using the default model, `gpt-4-1106-preview`, as it performs vastly better than any other model benchmarked so far. However, if you wish to use a different model, jump [here](#alternative-models).
-
 ### Add your OpenAI API Key
 
 You'll need to have API access to GPT-4 to run Mentat. There are a few options to provide Mentat with your OpenAI API key:
```
Comment (PCSwingle) [inline README.md:64]: Maybe this should be changed; everybody has gpt-4 access now, and it isn't even necessary anymore.

File: README.md [hunk 3]
```diff
@@ -70,66 +67,6 @@ You'll need to have API access to GPT-4 to run Mentat. There are a few options t
 2. Run `export OPENAI_API_KEY=<your key here>` prior to running Mentat
 3. Place the previous command in your `.bashrc` or `.zshrc` to export your key on every terminal startup
 
-### Azure OpenAI
```
Comment (PCSwingle) [inline README.md:73]: Is any of this information anywhere else? I think it's important to keep somewhere

Comment (jakethekoenig) [reply README.md:73]: It should all be here: https://docs.mentat.ai/en/latest/user/alternative_models.html

File: README.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 [![Twitter Follow](https://img.shields.io/twitter/follow/bio_bootloader?style=social)](https://twitter.com/bio_bootloader)
```
Comment (biobootloader) [inline README.md:1]:
```suggestion
[![Twitter Follow](https://img.shields.io/twitter/follow/AbanteAI?style=social)](https://twitter.com/AbanteAI)
```

**Review by biobootloader — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (PCSwingle) [reply README.md:73]: Could we link here here? I think it's important that it's easy for people to find the docs for alternative models

**Pull request merged**